tools/docker/builder/xenial/Dockerfile

```
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial
```

In the other one we call it ubuntu:18.04, so
```suggestion
FROM ubuntu:16.04
```